Tribler/Core/Libtorrent/LibtorrentMgr.py

```diff
@@ -78,11 +77,6 @@ def shutdown(self):
 
         self.get_session().stop_upnp()
 
-        # Save DHT state
```

Comment:
Hmm I'm not sure where the DHT state is saved now, I only know that it has been deprecated. Let me have a look in the documentation